### DISCUSSION TITLE
Welcome to Plus Toolkit Discussions!

### DISCUSSION THREAD

**Discussion by lassoan** [PlusToolkit/PlusLib#772] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by DevanshShukla41** [comment]
Hi Everyone!
We are working on a project in which we are using "Biim Portable Ultrasound probe," which uses a wifi connection (similar to Clarius Ultrasound Machine)  that connects the probe to their mobile application to get the Ultrasound images.

We are going to interface it with the PLUS toolkit for data collection. 

So for that, I wanted to ask you guys whether it will be possible to interface the PLUS toolkit for data collection with the "Biim Ultrasound Probe device," using the example which your lab has published on your website for Clarius Ultrasound Machine?

**Reply by adamrankin** [reply]
Does Biim provide an API for receiving images over the network that isn't their app?

**Reply by DevanshShukla41** [reply]
No Biim does not have an API for receiving the images over the network other than their mobile application, which is the problem.
But it does have an option in the application wherein we can basically input the server storage settings (Local AET Tittle; Remote AET Title; Server address; Port etc.) and send the dicom files on the server, and from there, I guess we could download them onto our PC'S. 
I have attached the screenshots from the application for it below.

I am currently working on setting up an open-source server (EasyPacs) that helps me host the dicom images.

Do you think that could work?

Thank You so much, and I appreciate all your help!

![Screenshot_2021-02-24-17-23-44-487_com biimultrasound BiimUltrasound](https://user-images.githubusercontent.com/79606934/109075199-d25f5f80-76c6-11eb-82f3-86d4037b9280.jpg)
![Screenshot_2021-02-24-17-23-50-886_com biimultrasound BiimUltrasound](https://user-images.githubusercontent.com/79606934/109075201-d2f7f600-76c6-11eb-85e9-9fce1a0d72db.jpg)

**Reply by lassoan** [reply]
I'll move this discussion to here: https://github.com/PlusToolkit/PlusLib/discussions/792
